src/components/DataCollector/index.js

```diff
@@ -33,7 +33,8 @@ const createDataCollector = ({ eventManager, logger }) => {
           viewStart = false,
           documentUnloading = false,
           type,
-          mergeId
+          mergeId,
+          scopes = []
```

Comment:
Can you please add an inline comment on what `scopes` are, and maybe a small example?

We need to add this info to the public docs. Should I create a ticket in the Alloy board, or you'd rather create on in Target's? @artur-ciocanu

Reply:
yes, sure